docs/ble.md

```diff
@@ -92,6 +89,48 @@ interface Characteristic {
 
 API Documentation
 -----------------
+BLE is an [EventEmitter](./events.md) with the following events:
+
+### Event: 'accept'
+
+* `string` `clientAddress`
+
+Emitted when a BLE client has connected. *NOTE*: currently the address is the
+dummy string "AB:CD:DF:AB:CD:EF".
```

Comment:
This is not true anymore, it now has a random fake client address generated from Zephyr.